viv_utils/__init__.py

```diff
@@ -58,6 +59,16 @@ def assertVwMatchesVivisectLibrary(vw):
         logger.debug("vivisect version match: %s", wanted)
 
 
+def loadWorkspaceFromViv(vw, viv_file, error):
+    if sys.version_info >= (3, 0):
+        try:
+            vw.loadWorkspace(viv_file)
+        except UnicodeDecodeError as e:
+            raise RuntimeError(error % viv_file) from e
```

Comment:
would you define and create a custom exception type here?

```
class IncompatibleVivVersion(ValueError): pass
```

```suggestion
            raise IncompatibleVivVersion("'%s' is an invalid .viv file. It may have been generated with Python 2 (incompatible with Python 3)." % viv_file) from e
```